pages/docs.css

```diff
@@ -237,253 +236,253 @@ body {
   flex-direction: column;
   gap: 0.5rem;
 }
-=======
-
-html, body {
-    font-family: Roboto, sans-serif;
-	overflow-anchor: none;
+======= html,
+body {
+  font-family: Roboto, sans-serif;
+  overflow-anchor: none;
 }
 
 .page-layout {
-    display: flex;
-    margin: auto;
-    overflow: clip;
-    overflow-x: hidden;
-    position: relative;
-    margin-top: 12%;
+  display: flex;
+  margin: auto;
+  overflow: clip;
+  overflow-x: hidden;
+  position: relative;
+  margin-top: 12%;
 }
 
 .drawer__sidebar--perm {
-    margin: 0;
-    margin-left: 0;
-    width: 20%;
-    height: 90vh;
-    position: fixed;
-    top: 20%;
-    box-shadow: var(--secondary-box-shadow);
-}
-    
+  margin: 0;
+  margin-left: 0;
+  width: 20%;
+  height: 90vh;
+  position: fixed;
+  top: 20%;
+  box-shadow: var(--secondary-box-shadow);
+}
+
 .drawer__headings {
-    padding: 1rem 1rem 0 1.5rem;
+  padding: 1rem 1rem 0 1.5rem;
 }
 
 .drawer__sidebar--perm h1 {
-    font-size: 2rem;
+  font-size: 2rem;
 }
 
 .drawer__sidebar--perm span {
-    color: var(--neutral-color);
+  color: var(--neutral-color);
 }
 
 .drawer__sidebar--perm ul {
-    overflow-y: auto;
-    margin: 2px;
-    list-style: none;
-    max-height: calc(100vh - 3rem);
-    overscroll-behavior: contain;
+  overflow-y: auto;
+  margin: 2px;
+  list-style: none;
+  max-height: calc(100vh - 3rem);
+  overscroll-behavior: contain;
 }
 
 .drawer__sidebar--perm ul li {
-    margin: .8rem 4px .8rem 1rem;
-    padding-left: .5rem;
-    border-radius: 4px;
-    font-size: 1.5rem;
+  margin: 0.8rem 4px 0.8rem 1rem;
+  padding-left: 0.5rem;
+  border-radius: 4px;
+  font-size: 1.5rem;
 }
 
 .drawer__sidebar--perm ul li:hover {
-    background-color: var(--accent-color);
+  background-color: var(--accent-color);
 }
 
 .active {
-    background-color: var(--secondary-bg-color);  
+  background-color: var(--secondary-bg-color);
 }
 
 .active > a .icon i {
-    color: var(--primary-color);  
+  color: var(--primary-color);
 }
 
 .drawer__sidebar--perm ul li.active:hover {
-    background-color: var(--secondary-bg-color);  
+  background-color: var(--secondary-bg-color);
 }
 
 .icon {
-    margin-right: 2rem;
-    font-size: 1.5rem;
+  margin-right: 2rem;
+  font-size: 1.5rem;
 }
 
 .labels {
-    display: inline-block;
-    margin: .5rem 2rem .5rem 2rem;
+  display: inline-block;
+  margin: 0.5rem 2rem 0.5rem 2rem;
 }
 
 .drawer__sidebar--perm ul li a {
-    display: flex;
-    align-items: center;
-    text-decoration: none;
-    font-size: 1rem;
-    line-height: 1.375rem;
-    font-weight: 600;
-    padding: .5rem 4px;
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  font-size: 1rem;
+  line-height: 1.375rem;
+  font-weight: 600;
+  padding: 0.5rem 4px;
```

Comment:
Unit consistency. If you are using rem, then use rem at all places. Change px here to rem.